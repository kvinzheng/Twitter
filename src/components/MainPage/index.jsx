import React from "react";

import LeftMenu from "../LeftMenu";
import RightMenu from "../RightMenu";
import CenterMenu from "../CenterMenu";

function MainPage() {
  return (
    <div className="MainPage">
      <LeftMenu />
      <CenterMenu />
      <RightMenu />
    </div>
  );
}
export default MainPage;
