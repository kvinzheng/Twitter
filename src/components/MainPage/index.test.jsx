import React from "react";

import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import MainPage from "./index";

configure({ adapter: new Adapter() });

describe("MainPage", () => {
  it("should render without any data defined", () => {
    const component = shallow(<MainPage />);

    expect(component).toMatchSnapshot();
  });
  it("Renders with correct classes", () => {
    const component = shallow(<MainPage />);
    expect(component.find(".MainPage").length).toBe(1);
  });
});
