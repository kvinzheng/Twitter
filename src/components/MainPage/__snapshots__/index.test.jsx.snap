// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage should render without any data defined 1`] = `ShallowWrapper {}`;
